pkg/steampipeconfig/modconfig/parsed_name.go

```diff
@@ -11,6 +11,11 @@ type ParsedResourceName struct {
 	Name     string
 }
 
+// ParseResourceName splits up a resource name into it's components
```

Comment:
lovely - but I actually meant an example for BuildFullResourceName :)

Reply:
pushed